File: app/src/main/java/com/maf/assignment/newsapp/network/responseModel/SourceData.java
package com.maf.assignment.newsapp.network.responseModel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by Ritesh on 04/09/2019.
 */

public class SourceData implements Serializable {

    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("name")
    @Expose
    private String name;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }


}

File: app/src/main/java/rk/information/news/network/responseModel/SourceData.kt
package rk.information.news.network.responseModel

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

/**
 * Created by Ritesh on 04/09/2019.
 */

class SourceData : Serializable {

    @SerializedName("id")
    @Expose
    var id: String? = null
    @SerializedName("name")
    @Expose
    var name: String? = null


}
